### ISSUE TITLE
[BUG] Unable to use Starlette 0.14.x

### ISSUE THREAD

**Issue by iwankgb** [schemathesis/schemathesis#1160]
**Describe the bug**
It is impossible to install Fastapi 0.65.1 and Schemathesis 3.6.7 in single environment. This is necessary because Fastapi 0.65.1 includes a Pedantic update that patches [CVE-2021-29510](https://github.com/samuelcolvin/pydantic/security/advisories/GHSA-5jqp-qgf6-3pvh).

**To Reproduce**
1. Run this command: `pip install fastapi==0.65.1 schemathesis==3.6.7`
2. See error
```
ERROR: Cannot install fastapi==0.65.1 and schemathesis==3.6.7 because these package versions have conflicting dependencies.

The conflict is caused by:
    fastapi 0.65.1 depends on starlette==0.14.2
    schemathesis 3.6.7 depends on starlette<0.14.0 and >=0.13.2
```

**Expected behavior**
Installation succeeds.

**Environment (please complete the following information):**
 - OS: macOS Big Sur
 - Python version: 3.9.4
 - Schemathesis version: 3.6.7
 - Spec version: irrelevant

**Additional context**
None.

**Comment by Stranger6667** [comment]
Thanks for reporting. Will fix soon

**Comment by Stranger6667** [comment]
The fix is released in `3.6.8`

**Comment by iwankgb** [comment]
@Stranger6667 you are my hero! <3
